sites/docs/content/components/dropdown-menu.md

```diff
@@ -16,4 +16,202 @@ description: Displays a menu of items that users can select from when triggered.
 
 </ComponentPreviewV2>
 
+## Structure
+
+```svelte
+<script lang="ts">
+	import { DropdownMenu } from "bits-ui";
+</script>
+
+<DropdownMenu.Root>
+	<DropdownMenu.Trigger />
+
+	<DropdownMenu.Content>
+		<DropdownMenu.Label />
```

Comment:
```suggestion
		<DropdownMenu.Group>
		<DropdownMenu.GroupLabel />
```

In previous versions of this, we weren't appropriately requiring a label to be associated with a group, which is confusing. In Svelte 5, if you're using a `GroupLabel`, it must be within either a `DropdownMenu.Group` or `DropdownMenu.RadioGroup`.